python/tvm/target/target.py

```diff
@@ -534,8 +534,14 @@ def hexagon(cpu_ver="v66", **kwargs):
         error if invalid. Does not affect codegen.
     llvm_options : str or list of str (default: None)
         User defined compiler arguments.
+    use_qfloat : bool (default: True for cpu_ver >= v68, False otherwise)
```

Comment:
Can we elaborate a bit here on the difference between / benefit of QFloat HVX and / over IEEE HVX instructions as after this patch will be defaulting to them for current and future archs?

Reply:
QFloat is faster and more power efficient, and is the default.  There may be some small precision differences in the outputs (plus values like infinity or NaN are not supported).